docs/book/products.rst

```diff
@@ -4,69 +4,161 @@
 Products
 ========
 
-**Product** model represents unique products in your Sylius store. Every product can have different variations or attributes and has following values:
+**Product** model represents unique products in your Sylius store.
+Every product can have different variations and attributes.
 
-* *name* - The full name of the product
-* *slug* - Urlized version of the name
-* *description* - Description of the product
-* *shortDescription* - Simple description of the product for lists and banners
-* *metaDescription* - Description for search engines
-* *metaKeywords* - SEO keywords
-* *createdAt* - Date of creation
-* *updateAt* - Timestamp of the most recent update
+.. warning::
+
+   Each product has to have at least one variant to be sold in the shop.
+
+How to create a Product?
+------------------------
+
+Before we learn how to create products that can be sold, let's see how to create a product without its complex dependencies.
+
+.. code-block:: php
+
+     /** @var ProductFactoryInterface $productFactory **/
+     $productFactory = $this->get('sylius.factory.product');
+
+     /** @var ProductInterface $product */
+     $product = $productFactory->createNew();
+
+Creating an empty product is not enough to save it in the database. It needs to have a ``name`` and ``code``.
+
+.. code-block:: php
+
+     $product->setName('T-Shirt');
+     $product->setCode('00001');
+
+     /** @var RepositoryInterface $productRepository */
+     $productRepository = $this->get('sylius.repository.product');
+
+     $productRepository->add($product);
+
+After being added via the repository, your product will be in the system. But the customers won't be able to buy it.
+
+Variants
+========
+
+**ProductVariant** represents a unique kind of product and can have its own pricing configuration, inventory tracking etc.
+
+Variants may be created out of Options of the product, but you are also able to use product variations system without the options at all.
+
+How to create a Product with a Variant?
```

Comment:
It can be done directly from the ProductFactory if you just call a `createWithVariant` function.